src/librawspeed/decoders/IiqDecoder.cpp

```diff
@@ -242,6 +257,47 @@ void IiqDecoder::DecodePhaseOneC(const std::vector<IiqStrip>& strips,
     DecodeStrip(strip, width, height);
 }
 
+void IiqDecoder::CorrectPhaseOneC(ByteStream meta_data, uint32 split_row,
+                                  uint32 split_col) {
+  ByteStream meta_data_base(meta_data.getSubStream(0));
+
+  meta_data.skipBytes(8);
+  const uint32 bytes_to_entries = meta_data.getU32();
+  meta_data.skipBytes(bytes_to_entries - 12);
```

Comment:
And if `bytes_to_entries` is less than `12`, you've got an unsigned overflow.
`12` is `8 + (32/4)`, i.e. `meta_data.getPos()`, so just `meta_data.setPos(bytes_to_entries);`